dataclients/kubernetes/clusterclient.go

```diff
@@ -438,7 +438,11 @@ func (c *clusterClient) loadEndpoints() (map[definitions.ResourceID]*endpoint, e
 	log.Debugf("all endpoints received: %d", len(endpoints.Items))
 	result := make(map[definitions.ResourceID]*endpoint)
 	for _, endpoint := range endpoints.Items {
-		result[endpoint.Meta.ToResourceID()] = endpoint
+		resID := endpoint.Meta.ToResourceID()
+		result[resID] = endpoint
+		if endpoint.Meta.Name == "skipper-ingress-redis" && endpoint.Meta.Namespace == "kube-system" {
+			log.Infof("clusterclient found addresses %d", len(endpoint.Subsets[0].Addresses))
+		}
```

Comment:
Why do we need this? Should it compare to configured `KubernetesRedisNamespace` and `KubernetesRedisName`?

Reply:
this is just my hack to see if it works at this place

Reply:
marked to remove before merge